src/frontend/src/apis/work-packages.api.ts

```diff
@@ -9,12 +9,12 @@ import { wbsPipe } from '../utils/pipes';
 import { apiUrls } from '../utils/urls';
 import { workPackageTransformer } from './transformers/work-packages.transformers';
 
-interface WorkPackageApiInputs {
+export interface WorkPackageApiInputs {
   name: string;
-  startDate: Date;
+  startDate: String;
   duration: number;
   crId: number;
-  stage: WorkPackageStage | null;
+  stage: WorkPackageStage | string | null;
```

Comment:
this shouldn't need to be string, look wherever the type mismatch is happening, you should be able to cast it as a WorkPackageStage

Reply:
I also fixed this